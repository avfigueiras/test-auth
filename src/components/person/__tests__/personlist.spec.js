import React from 'react';
import {render, act} from '@testing-library/react';
import ListPerson from '../index';
import axios from 'axios';

jest.mock('axios');

// Clean all promises
const flushPromises = () => new Promise(setImmediate);

describe("ListPerson tests", () => {
    test("ListPerson  test 1 with empty values", async () => {
        axios.get.mockImplementationOnce(() => Promise.resolve({data: {results: []}}));

        const ListPersonComponent = render(<ListPerson/>);

        await act(() => flushPromises());

        expect(ListPersonComponent).toMatchSnapshot();
        expect(axios.get).toHaveBeenNthCalledWith(1, 'https://randomuser.me/api/?results=100');
    });

    test("ListPerson test 1 with some values", async () => {
        const exampleUser = {
            "gender": "female",
            "name": {
                "title": "Miss",
                "first": "Manon",
                "last": "Mercier"
            },
            "location": {
                "street": {
                    "number": 6568,
                    "name": "Rue Abel-Gance"
                },
                "city": "CrÃ©teil",
                "state": "Loiret",
                "country": "France",
                "postcode": 26811,
                "coordinates": {
                    "latitude": "-2.0184",
                    "longitude": "169.0919"
                },
                "timezone": {
                    "offset": "-3:30",
                    "description": "Newfoundland"
                }
            },
            "email": "manon.mercier@example.com",
            "login": {
                "uuid": "aa44a549-2cf4-4a79-ba48-141e8e7c7bf3",
                "username": "orangebutterfly352",
                "password": "devon",
                "salt": "V9qMCKTQ",
                "md5": "3ec307678275bbc461ac77594d88954b",
                "sha1": "e48f861b1b637ed921c458308e35fe9afe66719d",
                "sha256": "d55ecdeb6684ae0eb9f6864dadfdd1c5df490454873aca92890e679a1516f2fd"
            },
            "dob": {
                "date": "1950-06-03T04:29:02.829Z",
                "age": 70
            },
            "registered": {
                "date": "2018-06-17T21:38:38.921Z",
                "age": 2
            },
            "phone": "05-47-47-92-64",
            "cell": "06-11-87-25-86",
            "id": {
                "name": "INSEE",
                "value": "2NNaN92449944 92"
            },
            "picture": {
                "large": "https://randomuser.me/api/portraits/women/53.jpg",
                "medium": "https://randomuser.me/api/portraits/med/women/53.jpg",
                "thumbnail": "https://randomuser.me/api/portraits/thumb/women/53.jpg"
            },
            "nat": "FR"
        };

        axios.get.mockImplementationOnce(() => Promise.resolve({data: {results: [exampleUser]}}));

        const ListPersonComponent = render(<ListPerson/>);

        await act(() => flushPromises());

        expect(ListPersonComponent).toMatchSnapshot();
        expect(axios.get).toHaveBeenNthCalledWith(1, 'https://randomuser.me/api/?results=100');
    });
});