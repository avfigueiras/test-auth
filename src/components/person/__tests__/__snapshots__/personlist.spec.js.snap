// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListPerson tests ListPerson  test 1 with empty values 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="makeStyles-pagesNone-12"
      >
        <nav
          aria-label="pagination navigation"
          class="MuiPagination-root"
        >
          <ul
            class="MuiPagination-ul"
          >
            <li>
              <button
                aria-label="Go to previous page"
                class="MuiButtonBase-root MuiPaginationItem-root MuiPaginationItem-page Mui-disabled Mui-disabled"
                disabled=""
                tabindex="-1"
                type="button"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiPaginationItem-icon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M15.41 7.41L14 6l-6 6 6 6 1.41-1.41L10.83 12z"
                  />
                </svg>
              </button>
            </li>
            <li>
              <button
                aria-label="Go to next page"
                class="MuiButtonBase-root MuiPaginationItem-root MuiPaginationItem-page Mui-disabled Mui-disabled"
                disabled=""
                tabindex="-1"
                type="button"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiPaginationItem-icon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M10 6L8.59 7.41 13.17 12l-4.58 4.59L10 18l6-6z"
                  />
                </svg>
              </button>
            </li>
          </ul>
        </nav>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="makeStyles-pagesNone-12"
    >
      <nav
        aria-label="pagination navigation"
        class="MuiPagination-root"
      >
        <ul
          class="MuiPagination-ul"
        >
          <li>
            <button
              aria-label="Go to previous page"
              class="MuiButtonBase-root MuiPaginationItem-root MuiPaginationItem-page Mui-disabled Mui-disabled"
              disabled=""
              tabindex="-1"
              type="button"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiPaginationItem-icon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M15.41 7.41L14 6l-6 6 6 6 1.41-1.41L10.83 12z"
                />
              </svg>
            </button>
          </li>
          <li>
            <button
              aria-label="Go to next page"
              class="MuiButtonBase-root MuiPaginationItem-root MuiPaginationItem-page Mui-disabled Mui-disabled"
              disabled=""
              tabindex="-1"
              type="button"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiPaginationItem-icon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M10 6L8.59 7.41 13.17 12l-4.58 4.59L10 18l6-6z"
                />
              </svg>
            </button>
          </li>
        </ul>
      </nav>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`ListPerson tests ListPerson test 1 with some values 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="makeStyles-rootMobile-46"
      >
        <div
          class="makeStyles-panelMobile-48"
        >
          <div
            class="MuiPaper-root MuiExpansionPanel-root makeStyles-panel-47 MuiExpansionPanel-rounded MuiPaper-elevation1 MuiPaper-rounded"
          >
            <div
              aria-disabled="false"
              aria-expanded="false"
              class="MuiButtonBase-root MuiExpansionPanelSummary-root makeStyles-header-54"
              role="button"
              tabindex="0"
            >
              <div
                class="MuiExpansionPanelSummary-content"
              >
                <div
                  class="MuiAvatar-root MuiAvatar-circle makeStyles-image-49"
                  height="50px"
                  width="50px"
                >
                  <img
                    alt="avatar"
                    class="MuiAvatar-img"
                    src="https://randomuser.me/api/portraits/med/women/53.jpg"
                  />
                </div>
                <br />
                <span
                  class="makeStyles-name-55"
                >
                  Manon
                   
                  Mercier
                </span>
              </div>
              <div
                aria-disabled="false"
                aria-hidden="true"
                class="MuiButtonBase-root MuiIconButton-root MuiExpansionPanelSummary-expandIcon MuiIconButton-edgeEnd"
              >
                <span
                  class="MuiIconButton-label"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                    />
                  </svg>
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </div>
            </div>
            <div
              class="MuiCollapse-container MuiCollapse-hidden"
              style="min-height: 0px;"
            >
              <div
                class="MuiCollapse-wrapper"
              >
                <div
                  class="MuiCollapse-wrapperInner"
                >
                  <div
                    role="region"
                  >
                    <div
                      class="MuiExpansionPanelDetails-root"
                    >
                      <div
                        class="makeStyles-generalMobile-59"
                      >
                        <h5>
                          <svg
                            aria-hidden="true"
                            class="MuiSvgIcon-root makeStyles-icon-53"
                            focusable="false"
                            viewBox="0 0 24 24"
                          >
                            <path
                              d="M14 2H6c-1.1 0-1.99.9-1.99 2L4 20c0 1.1.89 2 1.99 2H18c1.1 0 2-.9 2-2V8l-6-6zm2 16H8v-2h8v2zm0-4H8v-2h8v2zm-3-5V3.5L18.5 9H13z"
                            />
                          </svg>
                          Description:
                        </h5>
                        <hr
                          class="MuiDivider-root MuiDivider-middle"
                        />
                        <div
                          class="makeStyles-detailMobile-51"
                        >
                          <div>
                            <p
                              class="MuiTypography-root makeStyles-context-52 MuiTypography-body1"
                            >
                              <svg
                                aria-hidden="true"
                                class="MuiSvgIcon-root makeStyles-icon-53"
                                focusable="false"
                                viewBox="0 0 24 24"
                              >
                                <path
                                  d="M21 8V7l-3 2-3-2v1l3 2 3-2zm1-5H2C.9 3 0 3.9 0 5v14c0 1.1.9 2 2 2h20c1.1 0 1.99-.9 1.99-2L24 5c0-1.1-.9-2-2-2zM8 6c1.66 0 3 1.34 3 3s-1.34 3-3 3-3-1.34-3-3 1.34-3 3-3zm6 12H2v-1c0-2 4-3.1 6-3.1s6 1.1 6 3.1v1zm8-6h-8V6h8v6z"
                                />
                              </svg>
                               
                              manon.mercier@example.com
                            </p>
                            <p
                              class="MuiTypography-root makeStyles-context-52 MuiTypography-body1"
                            >
                              <svg
                                aria-hidden="true"
                                class="MuiSvgIcon-root makeStyles-icon-53"
                                focusable="false"
                                viewBox="0 0 24 24"
                              >
                                <path
                                  d="M10 20v-6h4v6h5v-8h3L12 3 2 12h3v8z"
                                />
                              </svg>
                               
                              Rue Abel-Gance
                               
                              6568
                              , 
                              Créteil
                              , 
                              France
                            </p>
                            <p
                              class="MuiTypography-root makeStyles-context-52 MuiTypography-body1"
                            >
                              <svg
                                aria-hidden="true"
                                class="MuiSvgIcon-root makeStyles-icon-53"
                                focusable="false"
                                viewBox="0 0 24 24"
                              >
                                <path
                                  d="M5.5 22v-7.5H4V9c0-1.1.9-2 2-2h3c1.1 0 2 .9 2 2v5.5H9.5V22h-4zM18 22v-6h3l-2.54-7.63C18.18 7.55 17.42 7 16.56 7h-.12c-.86 0-1.63.55-1.9 1.37L12 16h3v6h3zM7.5 6c1.11 0 2-.89 2-2s-.89-2-2-2-2 .89-2 2 .89 2 2 2zm9 0c1.11 0 2-.89 2-2s-.89-2-2-2-2 .89-2 2 .89 2 2 2z"
                                />
                              </svg>
                               
                              female
                            </p>
                            <p
                              class="MuiTypography-root makeStyles-context-52 MuiTypography-body1"
                            >
                              <svg
                                aria-hidden="true"
                                class="MuiSvgIcon-root makeStyles-icon-53"
                                focusable="false"
                                viewBox="0 0 24 24"
                              >
                                <path
                                  d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 3c1.66 0 3 1.34 3 3s-1.34 3-3 3-3-1.34-3-3 1.34-3 3-3zm0 14.2c-2.5 0-4.71-1.28-6-3.22.03-1.99 4-3.08 6-3.08 1.99 0 5.97 1.09 6 3.08-1.29 1.94-3.5 3.22-6 3.22z"
                                />
                              </svg>
                               
                              orangebutterfly352
                            </p>
                          </div>
                          <div>
                            <p
                              class="MuiTypography-root makeStyles-context-52 MuiTypography-body1"
                            >
                              <svg
                                aria-hidden="true"
                                class="MuiSvgIcon-root makeStyles-icon-53"
                                focusable="false"
                                viewBox="0 0 24 24"
                              >
                                <path
                                  d="M15.5 1h-8C6.12 1 5 2.12 5 3.5v17C5 21.88 6.12 23 7.5 23h8c1.38 0 2.5-1.12 2.5-2.5v-17C18 2.12 16.88 1 15.5 1zm-4 21c-.83 0-1.5-.67-1.5-1.5s.67-1.5 1.5-1.5 1.5.67 1.5 1.5-.67 1.5-1.5 1.5zm4.5-4H7V4h9v14z"
                                />
                              </svg>
                               
                              06-11-87-25-86
                            </p>
                            <p
                              class="MuiTypography-root makeStyles-context-52 MuiTypography-body1"
                            >
                              <svg
                                aria-hidden="true"
                                class="MuiSvgIcon-root makeStyles-icon-53"
                                focusable="false"
                                viewBox="0 0 24 24"
                              >
                                <path
                                  d="M6.62 10.79c1.44 2.83 3.76 5.14 6.59 6.59l2.2-2.2c.27-.27.67-.36 1.02-.24 1.12.37 2.33.57 3.57.57.55 0 1 .45 1 1V20c0 .55-.45 1-1 1-9.39 0-17-7.61-17-17 0-.55.45-1 1-1h3.5c.55 0 1 .45 1 1 0 1.25.2 2.45.57 3.57.11.35.03.74-.25 1.02l-2.2 2.2z"
                                />
                              </svg>
                               
                              05-47-47-92-64
                            </p>
                            <p
                              class="MuiTypography-root makeStyles-context-52 MuiTypography-body1"
                            >
                              <svg
                                aria-hidden="true"
                                class="MuiSvgIcon-root makeStyles-icon-53"
                                focusable="false"
                                viewBox="0 0 24 24"
                              >
                                <path
                                  d="M12 21.35l-1.45-1.32C5.4 15.36 2 12.28 2 8.5 2 5.42 4.42 3 7.5 3c1.74 0 3.41.81 4.5 2.09C13.09 3.81 14.76 3 16.5 3 19.58 3 22 5.42 22 8.5c0 3.78-3.4 6.86-8.55 11.54L12 21.35z"
                                />
                              </svg>
                               
                              70
                               years old
                            </p>
                            <p
                              class="MuiTypography-root makeStyles-context-52 MuiTypography-body1"
                            >
                              <svg
                                aria-hidden="true"
                                class="MuiSvgIcon-root makeStyles-icon-53"
                                focusable="false"
                                viewBox="0 0 24 24"
                              >
                                <path
                                  d="M16.5 12c1.38 0 2.49-1.12 2.49-2.5S17.88 7 16.5 7C15.12 7 14 8.12 14 9.5s1.12 2.5 2.5 2.5zM9 11c1.66 0 2.99-1.34 2.99-3S10.66 5 9 5C7.34 5 6 6.34 6 8s1.34 3 3 3zm7.5 3c-1.83 0-5.5.92-5.5 2.75V19h11v-2.25c0-1.83-3.67-2.75-5.5-2.75zM9 13c-2.33 0-7 1.17-7 3.5V19h7v-2.25c0-.85.33-2.34 2.37-3.47C10.5 13.1 9.66 13 9 13z"
                                />
                              </svg>
                               
                              2
                               years registered
                            </p>
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="makeStyles-pages-57"
      >
        <nav
          aria-label="pagination navigation"
          class="MuiPagination-root"
        >
          <ul
            class="MuiPagination-ul"
          >
            <li>
              <button
                aria-label="Go to previous page"
                class="MuiButtonBase-root MuiPaginationItem-root MuiPaginationItem-page Mui-disabled Mui-disabled"
                disabled=""
                tabindex="-1"
                type="button"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiPaginationItem-icon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M15.41 7.41L14 6l-6 6 6 6 1.41-1.41L10.83 12z"
                  />
                </svg>
              </button>
            </li>
            <li>
              <button
                aria-label="Go to next page"
                class="MuiButtonBase-root MuiPaginationItem-root MuiPaginationItem-page Mui-disabled Mui-disabled"
                disabled=""
                tabindex="-1"
                type="button"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiPaginationItem-icon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M10 6L8.59 7.41 13.17 12l-4.58 4.59L10 18l6-6z"
                  />
                </svg>
              </button>
            </li>
          </ul>
        </nav>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="makeStyles-rootMobile-46"
    >
      <div
        class="makeStyles-panelMobile-48"
      >
        <div
          class="MuiPaper-root MuiExpansionPanel-root makeStyles-panel-47 MuiExpansionPanel-rounded MuiPaper-elevation1 MuiPaper-rounded"
        >
          <div
            aria-disabled="false"
            aria-expanded="false"
            class="MuiButtonBase-root MuiExpansionPanelSummary-root makeStyles-header-54"
            role="button"
            tabindex="0"
          >
            <div
              class="MuiExpansionPanelSummary-content"
            >
              <div
                class="MuiAvatar-root MuiAvatar-circle makeStyles-image-49"
                height="50px"
                width="50px"
              >
                <img
                  alt="avatar"
                  class="MuiAvatar-img"
                  src="https://randomuser.me/api/portraits/med/women/53.jpg"
                />
              </div>
              <br />
              <span
                class="makeStyles-name-55"
              >
                Manon
                 
                Mercier
              </span>
            </div>
            <div
              aria-disabled="false"
              aria-hidden="true"
              class="MuiButtonBase-root MuiIconButton-root MuiExpansionPanelSummary-expandIcon MuiIconButton-edgeEnd"
            >
              <span
                class="MuiIconButton-label"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </div>
          </div>
          <div
            class="MuiCollapse-container MuiCollapse-hidden"
            style="min-height: 0px;"
          >
            <div
              class="MuiCollapse-wrapper"
            >
              <div
                class="MuiCollapse-wrapperInner"
              >
                <div
                  role="region"
                >
                  <div
                    class="MuiExpansionPanelDetails-root"
                  >
                    <div
                      class="makeStyles-generalMobile-59"
                    >
                      <h5>
                        <svg
                          aria-hidden="true"
                          class="MuiSvgIcon-root makeStyles-icon-53"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M14 2H6c-1.1 0-1.99.9-1.99 2L4 20c0 1.1.89 2 1.99 2H18c1.1 0 2-.9 2-2V8l-6-6zm2 16H8v-2h8v2zm0-4H8v-2h8v2zm-3-5V3.5L18.5 9H13z"
                          />
                        </svg>
                        Description:
                      </h5>
                      <hr
                        class="MuiDivider-root MuiDivider-middle"
                      />
                      <div
                        class="makeStyles-detailMobile-51"
                      >
                        <div>
                          <p
                            class="MuiTypography-root makeStyles-context-52 MuiTypography-body1"
                          >
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root makeStyles-icon-53"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M21 8V7l-3 2-3-2v1l3 2 3-2zm1-5H2C.9 3 0 3.9 0 5v14c0 1.1.9 2 2 2h20c1.1 0 1.99-.9 1.99-2L24 5c0-1.1-.9-2-2-2zM8 6c1.66 0 3 1.34 3 3s-1.34 3-3 3-3-1.34-3-3 1.34-3 3-3zm6 12H2v-1c0-2 4-3.1 6-3.1s6 1.1 6 3.1v1zm8-6h-8V6h8v6z"
                              />
                            </svg>
                             
                            manon.mercier@example.com
                          </p>
                          <p
                            class="MuiTypography-root makeStyles-context-52 MuiTypography-body1"
                          >
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root makeStyles-icon-53"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M10 20v-6h4v6h5v-8h3L12 3 2 12h3v8z"
                              />
                            </svg>
                             
                            Rue Abel-Gance
                             
                            6568
                            , 
                            Créteil
                            , 
                            France
                          </p>
                          <p
                            class="MuiTypography-root makeStyles-context-52 MuiTypography-body1"
                          >
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root makeStyles-icon-53"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M5.5 22v-7.5H4V9c0-1.1.9-2 2-2h3c1.1 0 2 .9 2 2v5.5H9.5V22h-4zM18 22v-6h3l-2.54-7.63C18.18 7.55 17.42 7 16.56 7h-.12c-.86 0-1.63.55-1.9 1.37L12 16h3v6h3zM7.5 6c1.11 0 2-.89 2-2s-.89-2-2-2-2 .89-2 2 .89 2 2 2zm9 0c1.11 0 2-.89 2-2s-.89-2-2-2-2 .89-2 2 .89 2 2 2z"
                              />
                            </svg>
                             
                            female
                          </p>
                          <p
                            class="MuiTypography-root makeStyles-context-52 MuiTypography-body1"
                          >
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root makeStyles-icon-53"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 3c1.66 0 3 1.34 3 3s-1.34 3-3 3-3-1.34-3-3 1.34-3 3-3zm0 14.2c-2.5 0-4.71-1.28-6-3.22.03-1.99 4-3.08 6-3.08 1.99 0 5.97 1.09 6 3.08-1.29 1.94-3.5 3.22-6 3.22z"
                              />
                            </svg>
                             
                            orangebutterfly352
                          </p>
                        </div>
                        <div>
                          <p
                            class="MuiTypography-root makeStyles-context-52 MuiTypography-body1"
                          >
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root makeStyles-icon-53"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M15.5 1h-8C6.12 1 5 2.12 5 3.5v17C5 21.88 6.12 23 7.5 23h8c1.38 0 2.5-1.12 2.5-2.5v-17C18 2.12 16.88 1 15.5 1zm-4 21c-.83 0-1.5-.67-1.5-1.5s.67-1.5 1.5-1.5 1.5.67 1.5 1.5-.67 1.5-1.5 1.5zm4.5-4H7V4h9v14z"
                              />
                            </svg>
                             
                            06-11-87-25-86
                          </p>
                          <p
                            class="MuiTypography-root makeStyles-context-52 MuiTypography-body1"
                          >
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root makeStyles-icon-53"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M6.62 10.79c1.44 2.83 3.76 5.14 6.59 6.59l2.2-2.2c.27-.27.67-.36 1.02-.24 1.12.37 2.33.57 3.57.57.55 0 1 .45 1 1V20c0 .55-.45 1-1 1-9.39 0-17-7.61-17-17 0-.55.45-1 1-1h3.5c.55 0 1 .45 1 1 0 1.25.2 2.45.57 3.57.11.35.03.74-.25 1.02l-2.2 2.2z"
                              />
                            </svg>
                             
                            05-47-47-92-64
                          </p>
                          <p
                            class="MuiTypography-root makeStyles-context-52 MuiTypography-body1"
                          >
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root makeStyles-icon-53"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M12 21.35l-1.45-1.32C5.4 15.36 2 12.28 2 8.5 2 5.42 4.42 3 7.5 3c1.74 0 3.41.81 4.5 2.09C13.09 3.81 14.76 3 16.5 3 19.58 3 22 5.42 22 8.5c0 3.78-3.4 6.86-8.55 11.54L12 21.35z"
                              />
                            </svg>
                             
                            70
                             years old
                          </p>
                          <p
                            class="MuiTypography-root makeStyles-context-52 MuiTypography-body1"
                          >
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root makeStyles-icon-53"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M16.5 12c1.38 0 2.49-1.12 2.49-2.5S17.88 7 16.5 7C15.12 7 14 8.12 14 9.5s1.12 2.5 2.5 2.5zM9 11c1.66 0 2.99-1.34 2.99-3S10.66 5 9 5C7.34 5 6 6.34 6 8s1.34 3 3 3zm7.5 3c-1.83 0-5.5.92-5.5 2.75V19h11v-2.25c0-1.83-3.67-2.75-5.5-2.75zM9 13c-2.33 0-7 1.17-7 3.5V19h7v-2.25c0-.85.33-2.34 2.37-3.47C10.5 13.1 9.66 13 9 13z"
                              />
                            </svg>
                             
                            2
                             years registered
                          </p>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="makeStyles-pages-57"
    >
      <nav
        aria-label="pagination navigation"
        class="MuiPagination-root"
      >
        <ul
          class="MuiPagination-ul"
        >
          <li>
            <button
              aria-label="Go to previous page"
              class="MuiButtonBase-root MuiPaginationItem-root MuiPaginationItem-page Mui-disabled Mui-disabled"
              disabled=""
              tabindex="-1"
              type="button"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiPaginationItem-icon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M15.41 7.41L14 6l-6 6 6 6 1.41-1.41L10.83 12z"
                />
              </svg>
            </button>
          </li>
          <li>
            <button
              aria-label="Go to next page"
              class="MuiButtonBase-root MuiPaginationItem-root MuiPaginationItem-page Mui-disabled Mui-disabled"
              disabled=""
              tabindex="-1"
              type="button"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiPaginationItem-icon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M10 6L8.59 7.41 13.17 12l-4.58 4.59L10 18l6-6z"
                />
              </svg>
            </button>
          </li>
        </ul>
      </nav>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
